db/ApplicationContext.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.EntityFrameworkCore;
+using TodoNET6.Models;
+
+namespace TodoNET6.db
+{
+    public class ApplicationContext : DbContext
+    {
+        public ApplicationContext(DbContextOptions options) 
+            : base(options)
```

Comment:
Is this saying ApplicationContext extends the base class of the options parameter?

And why is it needed / empty?

Reply:
This is how you pass the params that the base class requires to the base class, in this case `DbContext`. C# is very explicit in syntax so when you have an abstract class being implemented you have to be very clear with the compiler how the vase class' constructor is used.